openqaoa/optimizers/training_vqa.py

```diff
@@ -646,5 +648,160 @@ def optimize(self):
             raise e
         finally:
             self.results_dictionary()
+        
+        return self
+
+
+class PennyLaneOptimizer(OptimizeVQA):
+    """
+    Python custom scipy optimization with pennylane optimizers for the VQA class.
+
+    .. Tip::
+        Using bounds may result in lower optimization performance
+
+    Parameters
+    ----------
+    vqa_object:
+        Backend object of class VQABaseBackend which contains information on the backend used to perform computations, and the VQA circuit.
+    
+    variational_params:
+        Object of class QAOAVariationalBaseParams, which contains information on the circuit to be executed,  the type of parametrisation, and the angles of the VQA circuit.
+
+    optimizer_dict:
+        * jac
```

Comment:
Why so many white lines?

Reply:
I copied the CustomScipyGradientOptimizer class, but I think this documentation's format is incorrect. Should I change all of it?